locale/en/download/package-manager.md

```diff
@@ -237,6 +238,47 @@ from source:
 nvm uninstall 8
 ```
 
+## nvs
+While nvm is recommended for  Unix / OS X systems, and there is an unaffiliated `nvm-windows` version manager for Windows. The recommended version manager for Windows is `nvs`.
```

Comment:
The other entries avoid endorsement. This recommendation seems inappropriate.

Reply:
The below will be good?
"While nvm is recommended for  Unix / OS X systems, the recommended version manager for Windows is `nvs`."

Reply:
I changed the sentence to **"`nvs` is a version manager for Windows."**